API migration guides (#14575)
DataDog/documentation: content/en/security_platform/cspm/frameworks_and_benchmarks.md

For example, you can remove all resources tagged with `env:staging` using the **This rule will not generate a finding if there is a match with any of the following suppression queries** function. Or, limit the scope for a certain rule to resources tagged with `compliance:pci` using the **Only generate a finding if there is a match with any of the following queries** function.

{{< img src="security_platform/cspm/frameworks_and_benchmarks/never-trigger-a-finding.png" alt="In the Datadog app, select Advanced to populate Never trigger a finding when, and add a query." >}}

## Set notification targets for detection rules

From the [Rules][12] page, you can add notification targets. The complete list of notification options are:

- [Slack][13]
- [Jira][14]
- [PagerDuty][15]
- [ServiceNow][16]
- [Microsoft Teams][17]
- [Webhooks][18]
- Email

Set the severity of security posture findings. The dropdown allows you to select an appropriate severity level (INFO, LOW, MEDIUM, HIGH, CRITICAL).

> In the “Notify” section, configure zero or more [notification targets][9] for each rule case.

Reviewer: ```suggestion
In the “Notify” section, any [notification targets][9] for each rule case.
```
